src/components/MDX/ExpandableCallout.tsx

```diff
@@ -18,7 +18,7 @@ interface ExpandableCalloutProps {
 
 const variantMap = {
   deprecated: {
-    title: 'Deprecated',
+    title: 'Obsoleto',
```

Comment:
Por favor, ¿podrías revertir este cambio?
La recomendación del equipo de React es no traducir elementos de la UI de momento para evitar conflictos en el futuro:
> Keep in mind translating UI is best done closer to the end. We plan to make non-trivial code changes to the repo over the next few months. So it's best to focus on the content rather than on UI components.